lib/matplotlib/projections/polar.py

```diff
@@ -275,12 +270,12 @@ def zoom(self, direction):
 class ThetaTick(maxis.XTick):
     """
     A theta-axis tick.
-
+    
```

Comment:
Could these tabs turn back into empty blank lines? (so there shouldn't be any diff on the lines)